pkg/cloud/awsprovider.go

```diff
@@ -1517,6 +1526,14 @@ func (aws *AWS) GetAddresses() ([]byte, error) {
 	})
 }
 
+func (aws *AWS) isAddressOrphaned(address *ec2Types.Address) bool {
```

Comment:
I made a bunch of private methods in this PR, just want to call that out in case that has some weird implications

Reply:
No problem!